backend/substrapp/tests/tests_tasks.py

```diff
@@ -220,7 +220,7 @@ def test_prepare_data_sample_zip(self):
 
     def test_prepare_data_sample_zip_fail(self):
 
-        data_sample = DataSample(pkhash='foo', path=self.data_sample)
+        data_sample = DataSample(pkhash='foo', path=self.data_sample_filename)
```

Comment:
Why not using https://github.com/SubstraFoundation/substra-backend/blob/fc247da9557b69f241e6cfae653983e1d09f4fd3/backend/substrapp/tests/query/tests_query_datasample.py#L199:L201

Reply:
could you give me more details? a filename here seems to be sufficient

Reply:
A file name is a string, but not a path. It work as it will not raise, but it can be better to use a path to the file (even fake) to be consistent.